clang/lib/CIR/Dialect/Transforms/TargetLowering/TargetInfo.cpp

```diff
@@ -8,5 +8,11 @@ TargetLoweringInfo::TargetLoweringInfo(std::unique_ptr<ABIInfo> Info)
 
 TargetLoweringInfo::~TargetLoweringInfo() = default;
 
+AddressSpaceAttr::MapTy const &TargetLoweringInfo::getCIRAddrSpaceMap() const {
+  // Flatten all non-target address spaces to addrspace(0)
+  static AddressSpaceAttr::MapTy defaultAddrSpaceMap = {};
```

Comment:
Why does this need to be `static`? We usually reserve using local static variables only when it's strictly needed, and looks like this function could just be a `return {}`? Note that this or the target versions are not called much more than a couple times in a pipeline, so probably not worth any early optimization.